easybuild/easyconfigs/s/scVelo/scVelo-0.1.24-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,104 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'scVelo'
+version = '0.1.24'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://scvelo.org"
+description = """scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
+ dynamical modeling."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [('pkg-config', '0.29.2')]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('h5py', '2.10.0', versionsuffix),
+    ('networkx', '2.4', versionsuffix),
+    ('numba', '0.47.0', versionsuffix),
+    ('PyTables', '3.6.1', versionsuffix),
+    ('statsmodels', '0.11.0', versionsuffix),
+    ('libpng', '1.6.37'),
+    ('freetype', '2.10.1'),
+    ('Tkinter', '%(pyver)s'),
+    ('tqdm', '4.41.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('Cycler', '0.10.0', {
+        'modulename': 'cycler',
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+        'checksums': ['cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8'],
+    }),
+    ('kiwisolver', '1.1.0', {
+        'checksums': ['53eaed412477c836e1b9522c19858a8557d6e595077830146182225613b11a75'],
+    }),
+    ('matplotlib', '3.0.3', {
```

Comment:
why matplotlib as an extension instead of dependency? is it because of the version?

Reply:
Yes, it was due to _scanpy_, which had a `matplotlib==3.0.*` requirement – however, since submitting this PR they've now bumped to 3.1 (see https://github.com/theislab/scanpy/pull/1020). These changes haven't made it to a release yet but hopefully a scanpy patch will be relatively trivial, which would allow the existing `matplotlib-3.1.1-foss-2019b-Python-3.7.4.eb` to be used as a dependancy instead. Would you like me to do this?

Reply:
@orbsmiv it does seem to be better, if it is indeed a trivial patch

Reply:
Fixed in 6a70864